test/v2/services/ListPaymentsAndAllocationDetailsServiceSpec.scala

```diff
@@ -72,9 +72,10 @@ class ListPaymentsAndAllocationDetailsServiceSpec extends ServiceSpec {
         "INVALID_DATE_FROM"        -> FromDateFormatError,
         "INVALID_DATE_TO"          -> ToDateFormatError,
         "INVALID_DATE_RANGE"       -> RuleInvalidDateRangeError,
-        "REQUEST_NOT_PROCESSED"    -> InternalError,
+        "INVALID_REQUEST"          -> RuleInconsistentQueryParamsErrorListSA,
```

Comment:
I don't think the tech specs list "INVALID_REQUEST" as a backend error. Would this error scenario get covered by the frontend validation in any case? (e.g. MISSING_TO_DATE, MISSING_FROM_DATE, MISSING_PAYMENT_LOT, or MISSING_PAYMENT_LOT_ITEM)